app/ui/frontend/dist/about.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
interesting..`dist` would get generated by build process, no?

Reply:
I couldn't figure out how to get the webpack configs to pick up HTML files other than `index.html` when I run `npm run build`.

So my solution for now was to just directly stick them in there

Reply:
yeah, as long as the build does not `clean` stuff as the first step, we are fine..